packages/auth/src/access/types.ts

```diff
@@ -109,6 +109,7 @@ export function isAccessRequest(
   )
 }
 
+// value should hold bigint, serialized as string for requests & storage as jsonb field in postgres
```

Comment:
`jsonb` is a way that Postgres can store JSON payloads. Rather than storing a stringified form of the JSON like the `json` format does, `jsonb` decomposes it into a binary format so that it may be indexed.
https://www.postgresql.org/docs/current/datatype-json.html